code/modules/antagonists/changeling/changeling.dm

```diff
@@ -18,7 +18,7 @@
 
 	var/list/stored_profiles = list() //list of datum/changelingprofile
 	var/datum/changelingprofile/first_prof = null
-	var/dna_max = 6 //How many extra DNA strands the changeling can store for transformation.
+	var/dna_max = 99 //How many extra DNA strands the changeling can store for transformation.
```

Comment:
If you intend to essentially remove the dna strand cap, you should be more clear about it in your changelog, and It's bad practice to just leave in code that is not really neccesary, I suggest that you get rid of this var and the places it's used so we do not have more unused code floating around.
Not to mention that your changelog is suggesting you only added a couple more slots, when in reality you made it essentially unlimited.